content/organizing-faq.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Kubernetes Community Day FAQ 
+---
+
+# Are Kubernetes Community Days limited to Kubernetes?
+
+No! Any open source, cloud native technology is appropriate to cover at a KCD. The only requirements are that presentations not be vendor pitches. Please see the cloud native [definition](https://github.com/cncf/toc/blob/master/DEFINITION.md).
+
+# Can I call it a Cloud Native Day, Mini-KubeCon, or similar?
+
+Sorry, but it needs to be called Kubernetes Community Day [Location] (e.g., Kubernetes Community Day Bouvet Island). There are two reasons. First, we need it to include the term Kubernetes, as CNCF's ownership of the trademark lets us maintain control. Specifically, cloud native is not trademarked so anyone can run a Cloud Native Day, but Kubernetes Community Days can only operate under CNCF's aegis. Second, we believe that there will be a lot of value over time in building up the Kubernetes Community Day brand, in that it will enable new cities to more easily recruit speakers, sponsors, and attendees.
+
+# Why is CNCF mandating the use of specific tools like Eventbrite for registration, Sched for the schedule, and SM Apply for the CFP?
+
```

Comment:
Should we also add a question specifically about Eventbrite and how to use it? It might make more sense to add this to the organizing page but we received a few questions about it. My goal is to clear up that organizers will use CNCF's Eventbrite account and other details on how to get started.

Reply:
Yes, but we need to get Eventbrite set up first! Probably worth adding later.